assets/scripts/utils/html.js

```diff
@@ -1,120 +1,144 @@
-    const data = {};
+    const data = {}
 
     for (let i in attributes) {
         if (!attributes[i]) {
-            continue;
+            continue
         }
 
         // Attributes name (ex: data-module)
-        let name = attributes[i].name;
+        let name = attributes[i].name
 
         // This happens.
         if (!name) {
-            continue;
+            continue
         }
 
-        let match = name.match(pattern);
+        let match = name.match(pattern)
         if (!match) {
-            continue;
+            continue
         }
 
         // If this throws an error, you have some
         // serious problems in your HTML.
-        data[match[1]] = getData(node.getAttribute(name));
+        data[match[1]] = getData(node.getAttribute(name))
     }
 
     return data;
+
 }
 
-const rbrace = /^(?:\{[\w\W]*\}|\[[\w\W]*\])$/;
+
+
 
 /**
  * Parse value to data type.
  *
  * @link   https://github.com/jquery/jquery/blob/3.1.1/src/data.js
- * @param  {string} data - A value to convert.
- * @return {mixed}  Returns the value in its natural data type.
+ * @param  {string} data - value to convert
+ * @return {mixed}  value in its natural data type
  */
-export function getData(data) {
+
+const rbrace = /^(?:\{[\w\W]*\}|\[[\w\W]*\])$/
+const getData = data => {
     if (data === 'true') {
-        return true;
+        return true
     }
 
     if (data === 'false') {
-        return false;
+        return false
     }
 
     if (data === 'null') {
-        return null;
+        return null
     }
 
     // Only convert to a number if it doesn't change the string
     if (data === +data+'') {
-        return +data;
+        return +data
     }
 
-    if (rbrace.test( data )) {
-        return JSON.parse( data );
+    if (rbrace.test(data)) {
+        return JSON.parse(data)
     }
 
-    return data;
+    return data
 }
 
+
 /**
  * Returns an array containing all the parent nodes of the given node
- * @param  {object} node
- * @return {array} parent nodes
+ * @param  {HTMLElement}    $el     - DOM Element
+ * @return {array}          parent nodes
  */
-export function getParents(elem) {
+
+const getParents = $el => {
+
     // Set up a parent array
-    let parents = [];
+    let parents = []
 
     // Push each parent element to the array
-    for ( ; elem && elem !== document; elem = elem.parentNode ) {
-        parents.push(elem);
+    for (; $el && $el !== document; $el = $el.parentNode) {
+        parents.push($el)
     }
 
     // Return our parent array
-    return parents;
+    return parents
 }
 
-// https://gomakethings.com/how-to-get-the-closest-parent-element-with-a-matching-selector-using-vanilla-javascript/
-export function queryClosestParent(elem, selector) {
+
+/**
+ * Find every closest parent to element
+ * @param  {HTMLElement}    $el     - DOM Element
+ * @param  {string}         select  - parent elements seelctor
+ * @return {array}          parent nodes
+ */
+
+const queryClosestParent = ($el, selector) => {
```

Comment:
Either remove this function in favour of `Element.closest()` or have it be an alias of it.

I prefer we remove this custom function.

Reply:
@lucasvallenet 👋